openedx/core/djangoapps/profile_images/images.py

```diff
@@ -0,0 +1,128 @@
+"""
+Image file manipulation functions related to profile images.
+"""
+from cStringIO import StringIO
+
+from django.conf import settings
+from django.core.files.base import ContentFile
+from PIL import Image
+
+from ..user_api.accounts.api import get_profile_image_storage
+
+
+class DevMsg(object):
+    """
+    Holder for pseudo-constants.
+    """
+    FILE_TOO_LARGE = 'Maximum file size exceeded.'
```

Comment:
hm.  @cahrens do you think we should just ditch the developer messages, make these user messages, and mark them for translation?  Seems weird to maintain both when they'll essentially carry identical information.